Dockerfile

```
@@ -1,5 +1,5 @@
 # Freebind build
-FROM debian:buster-slim as freebind
+FROM debian:bookworm-slim as freebind
```

Nitpick, as long as we're updating this line
```suggestion
FROM debian:bookworm-slim AS freebind
```